src/_negative-margins.css

```diff
@@ -0,0 +1,199 @@
+/*
+   NEGATIVE MARGINS
+
+   Base:
+     n = negative
+
+   Modifiers:
+     a = all
+     t = top
+     r = right
+     b = bottom
+     l = left
+
+     1 = 1st step in spacing scale
+     2 = 2nd step in spacing scale
+     3 = 3rd step in spacing scale
+     4 = 4th step in spacing scale
+     5 = 5th step in spacing scale
+     6 = 6th step in spacing scale
+     7 = 7th step in spacing scale
+
+   Media Query Extensions:
+     -ns = not-small
+     -m  = medium
+     -l  = large
+
+*/
+
+
+
+.na1 {  margin: -var(--spacing-extra-small); }
+.na2  {  margin: -var(--spacing-small); }
+.na3  {  margin: -var(--spacing-medium); }
+.na4  {  margin: -var(--spacing-large); }
+.na5  {  margin: -var(--spacing-extra-large); }
+.na6 {  margin: -var(--spacing-extra-extra-large); }
+.na7 { margin: -var(--spacing-extra-extra-extra-large); }
```

Comment:
Is the alignment in these blocks — with the extra whitespace between the 2-5 class names and the `{` — intentional?